generators/client/templates/react/src/test/javascript/spec/app/modules/administration/administration.reducer.spec.ts.ejs

```diff
@@ -0,0 +1,213 @@
+import { expect } from 'chai';
+import { REQUEST, FAILURE, SUCCESS } from 'app/shared/reducers/action-type.util';
+
+import administration, { ACTION_TYPES } from 'app/modules/administration/administration.reducer';
+
+describe('User managment reducer tests', () => {
```

Comment:
user management is on its own reducer so the test for that should be moved to `administration/user-management` folder

Reply:
Also look at the `generator-jhipster/generators/entity-client/templates/react/src/test/javascript/spec/app/entities/entity-reducer.spc.ts.ejs` to see how the actions can be tested as well